src/components/gearGrid.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {Modal, Button} from 'react-bootstrap';
+import Selector, {WeaponCard} from './shared/card';
+import './builder.scss'
+import Stats from './stats.js'
+
+class GearSelection extends React.Component {
+    state={
```

Comment:
does this work? I thought you had to wrap it in a constructor

Reply:
You can as long as you don't need props. This will need to be updated when we adapt to use it all linked together. 

> The question to answer: How to set initial state in React without a constructor? In this case, there is an initial state for the toggle property in the App component. By using an alternative class syntax, you can leave out the constructor and initialize the state as class field declaration. However, you don't have access to the props anymore.

https://www.robinwieruch.de/react-state-without-constructor